porch/func/wrapper/wrapper.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 Google LLC
```

Comment:
File naming nit: maybe name similarly to the pod, i.e. `wrapper-server` as a directory name or file name.

Reply:
Done